content/en/docs/tasks/observability/distributed-tracing/overview/index.md

```diff
@@ -12,60 +12,48 @@ test: n/a
+
+## Extension providers
+
+[Extension providers](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ExtensionProvider) are defined in `MeshConfig`, and allow defining the configuration for a trace backend. Supported providers are OpenTelemetry, Zipkin, SkyWalking, Datadog and Stackdriver.
+
+## Building applications to support trace context propagation
 
 Although Istio proxies can automatically send spans, extra information is needed to join those spans into a single trace. Applications must propagate this information in HTTP headers, so that when proxies send spans, the backend can join them together into a single trace.
 
 To do this, each application must collect headers from each incoming request and forward the headers to all outgoing requests triggered by that incoming request. The choice of headers to forward depends on the configured trace backend. The set of headers to forward are described in each trace backend-specific task page. The following is a summary:
 
-All applications should forward the following header:
+All applications should forward the following headers:
 
-* `x-request-id`: this is an envoy-specific header that is used to consistently sample logs and traces.
+* `x-request-id`: an Envoy-specific header that is used to consistently sample logs and traces.
+* `traceparent` and `tracestate`: [W3C standard headers](https://www.w3.org/TR/trace-context/)
 
-For Zipkin, Jaeger, and Stackdriver the B3 multi-header format should be forwarded:
+For Zipkin, the [B3 multi-header format](https://github.com/openzipkin/b3-propagation) should be forwarded:
 
 * `x-b3-traceid`
 * `x-b3-spanid`
 * `x-b3-parentspanid`
 * `x-b3-sampled`
 * `x-b3-flags`
 
-These are supported by Zipkin, Jaeger, and many other tools.
-
-For Datadog, the following headers should be forwarded. Forwarding these is handled automatically by Datadog client libraries for many languages and frameworks.
+For commercial observability tools, refer to their documentation.
 
-* `x-datadog-trace-id`.
-* `x-datadog-parent-id`.
-* `x-datadog-sampling-priority`.
-
-For Lightstep, the OpenTracing span context header should be forwarded:
-
-* `x-ot-span-context`
-
-For Stackdriver, you can choose to use any one of the following headers instead of the B3 multi-header format.
-
-* `grpc-trace-bin`: Standard grpc trace header.
-* `traceparent`: W3C Trace Context standard for tracing. Supported by OpenTelemetry and an increasing number of Jaeger client libraries.
-* `x-cloud-trace-context`: used by Google Cloud product APIs.
-
-If you look at the sample Python `productpage` service, for example, you see that the application extracts the required headers for all tracers from an HTTP request using [OpenTracing](https://opentracing.io/) libraries:
+If you look at the sample Python `productpage` service, for example, you see that the application extracts the required headers for all tracers from an HTTP request using OpenTelemetry libraries:
```

Comment:
Can we add a link to the `productpage` line of code to make it easier for users to navigate to it

Reply:
Note to future self, https://github.com/istio/istio/blob/master/samples/bookinfo/src/productpage/productpage.py#L125

Reply:
Done!